mrcnn/model.py

```diff
@@ -2198,14 +2222,16 @@ def build(self, mode, config):
                        mrcnn_class_logits, mrcnn_class, mrcnn_bbox, mrcnn_mask,
                        rpn_rois, output_rois,
                        rpn_class_loss, rpn_bbox_loss, class_loss, bbox_loss, mask_loss]
+            if config.NUm_IMAGe_CLASSES:
+                outputs.append(image_classes)
```

Comment:
I think we should place that output before the loss outputs. 

Also, I believe we need an additional loss output for `image_class` here explicitly. (See other comment about `add_loss` below.)

Reply:
Done.